corehq/motech/requests.py

```diff
@@ -132,7 +131,7 @@ def _send_request(self, method, url, *args, **kwargs):
         if not self.verify:
             kwargs['verify'] = False
         kwargs.setdefault('timeout', REQUEST_TIMEOUT)
-        sanitize_user_input_url_for_repeaters(url, domain=self.domain_name, src='sent_attempt')
+        validate_user_input_url_for_repeaters(url, domain=self.domain_name, src='sent_attempt')
```

Comment:
Since this now raises `CannotResolveHost` and it didn't before, could this result in a change in how the user sees the error in the UI? I can't tell from this, but I want to make sure you considered that

Reply:
Specifically for the _Test Connection_ button, I verified locally that the user sees: `
Failed! Response is: Unknown URL`. From a broader perspective, this only raises `CannotResolveHost` if the host isn't resolvable. If the host isn't resolvable, the existing code would raise an exception on the very next statement (`session.request(...)`) anyway. Unless code is specifically catching `session.request`'s error, then the result should be the same. Looking over how incremental exports handle this, it doesn't seem like it handles errors in this process at all.